include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
Clearly you feel differently about it, but I think that "wanting to include StandardCLibrary.h" *is* the real problem. :disappointed_relieved:   (It's an implementation detail, not a public header.)